drivers/flash/Kconfig

```diff
@@ -36,6 +36,17 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config FLASH_EMULATOR
```

Comment:
lest create sub menu for flash emulator: 'menuconfig FLASH_EMULATOR`. All entries dependent on FLASH_EMULATOR will land in this submenu.